cypress/e2e/home_spec.cy.js

```diff
@@ -7,7 +7,7 @@ describe('As a user, when I load the application, I can see a collection of pain
     })
```

Comment:
Hey @Mlberry0205 , can you explain what the .then in line 5 does? I looked it up in the docs and am not understanding its role

Reply:
Hey Alycia, The .then() is was waiting for our data to return before displaying it.